lib/util/problem.js

```diff
@@ -181,9 +187,7 @@ const problems = {
 
     editingClosingOrClosed: problem(409.12, () => 'You are trying to edit a submission of a Form that is in Closing or Closed state. In this version of Central, you can only edit the submissions of Open Forms. We will fix this in a future version for Central.'),
 
-    enketoEditRateLimit: problem(409.13, () => 'You must wait one minute before trying to edit a submission a second time.'),
-
-    entityViolation: problem(409.14, ({ reason }) => `There was a problem with entity processing: ${reason}`)
```

Comment:
409.14 is already in production, right? If so, I think we should leave a comment here that 409.14 was previously used, but has been deprecated. That way, we'll know not to reuse that Problem code for a different type of Problem.